.gitlab-ci.yml

```diff
@@ -190,3 +192,28 @@ force_release_prod:
   only:
     - staging
 
+verify_sync_partial:on-schedule:
+  stage: verify_sync
+  only:
+    refs:
+      - schedules
+  before_script:
+    - VERIFY_SECRET_NAME="${DSS_SECRETS_STORE}/${DSS_DEPLOYMENT_STAGE}/${DSS_VERIFY_SECRETS_NAME}"
+    - LAST_VERIFIED="$(python scripts/dss-ops.py secrets get --secret-names ${VERIFY_SECRET_NAME} --json | jq -r '.[]')"
```

Comment:
```suggestion
    - LAST_VERIFIED="$(python scripts/dss-ops.py secrets get ${VERIFY_SECRET_NAME})"
```

Reply:
There was a PR recently that updated/improved the secrets handling in dss-ops. It did away with the `--secret-names` flag, and values can only be piped in or provided via an input file.

Reply:
Thanks!